src/ddmd/toctype.d

```diff
@@ -170,14 +170,24 @@ public:
         //printf("TypeEnum::toCtype() '%s'\n", t.sym.toChars());
         if (t.mod == 0)
         {
+            auto baseType = t.sym.memtype.toBasetype();
             if (!t.sym.memtype)
```

Comment:
Not a good place to dereference memtype, considering this condition.

Reply:
Ok, how about now?

Reply:
> And as per @MartinNowak comments, needs test in testsuite.

Added.